src/main/java/io/confluent/connect/elasticsearch/Mapping.java

```diff
@@ -47,14 +47,6 @@
 
   private static final Logger log = LoggerFactory.getLogger(Mapping.class);
 
-  /**
-   * Create an explicit mapping.
-   * @param client The client to connect to Elasticsearch.
-   * @param index The index to write to Elasticsearch.
-   * @param type The type to create mapping for.
-   * @param schema The schema used to infer mapping.
-   * @throws IOException
-   */
```

Comment:
Why is the javadoc removed? I would expect it to expand to cover all public methods.